homeassistant/components/arduino/sensor.py

```diff
@@ -14,8 +15,16 @@
 
 CONF_PINS = "pins"
 CONF_TYPE = "analog"
```

Comment:
We only use `CONF_` for config keys, not for config values. Please rename this constant to eg `ANALOG_MODE`.